File: DuDoong-Domain/src/main/java/band/gosrock/domain/domains/order/exception/NotOwnerOrderException.java
package band.gosrock.domain.domains.order.exception;


import band.gosrock.common.exception.DuDoongCodeException;
import band.gosrock.common.exception.ErrorCode;

public class NotOwnerOrderException extends DuDoongCodeException {

    public static final DuDoongCodeException EXCEPTION = new NotOwnerOrderException();

    private NotOwnerOrderException() {
        super(OrderErrorCode.ORDER_NOT_MINE);
    }
}

File: DuDoong-Domain/src/main/kotlin/band/gosrock/domain/domains/order/exception/NotOwnerOrderException.kt
package band.gosrock.domain.domains.order.exception

import band.gosrock.common.exception.DuDoongCodeException

class NotOwnerOrderException private constructor() : DuDoongCodeException(OrderErrorCode.ORDER_NOT_MINE) {
    companion object {
        @JvmField
        val EXCEPTION: DuDoongCodeException = NotOwnerOrderException()
    }
}
